<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilityWidget</class>
 <widget class="QWidget" name="UtilityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnMaintain">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>系统维护</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="btnStyle" stdset="0">
           <string>normalBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSystemSet">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>系统设置</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="btnStyle" stdset="0">
           <string>normalBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnItemSet">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>项目设置</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="btnStyle" stdset="0">
           <string>normalBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnState">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>状态</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="btnStyle" stdset="0">
           <string>normalBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnASSetting">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>辅助设置</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="btnStyle" stdset="0">
           <string>normalBtn</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageMantain">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTabWidget" name="tabMantain"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSystemSet">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTabWidget" name="tabSystemSet"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageItemSet">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTabWidget" name="tabItemSet"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageState">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTabWidget" name="tabState"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageASSetting">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTabWidget" name="tabASSetting"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
